docs/content/sftp.md

```diff
@@ -107,10 +107,20 @@ The SFTP remote supports three authentication methods:
 Key files should be PEM-encoded private key files. For instance `/home/$USER/.ssh/id_rsa`.
 Only unencrypted OpenSSH or PEM encrypted files are supported.
 
-If you don't specify `pass` or `key_file` then rclone will attempt to contact an ssh-agent.
+The key file can be specified in either an external file (key_file) or contained within the 
+rclone config file (key_pem).  If using key_pem in the config file, the entry should be on a
+single line with new line ('\n','\r\n') separating lines.  i.e. 
```

Comment:
Your awk doesn't replace `\r`... Is `\r` likely? I guess it might be on Windows

Reply:
Yes.  Thats better.   Thanks!